internal/kibana/packages.go

```diff
@@ -41,6 +41,7 @@ func (c *Client) InstallZipPackage(zipFile string) ([]packages.Asset, error) {
 		return nil, err
 	}
 	req.Header.Set("Content-Type", "application/zip")
+	req.Header.Add("elastic-api-version", "2023-10-31")
```

Comment:
Should this header be added for all other requests performed by elastic-package (dashboards, status...)?

Maybe, it can be added (or moved?) here for all of them:
https://github.com/elastic/elastic-package/blob/689e77a11c2f534aad75f3f1f3bdc7dd375eee72/internal/kibana/client.go#L153

Reply:
Not sure, it only seems to be required here, maybe this API has been GA'ed on this version?

Reply:
Ok. I guess we could keep it here, and move it to the `newRequest()` function if there are new other API requests failing with the same error.